raiden/network/rpc/client.py

```diff
@@ -558,7 +558,7 @@ def deploy_single_contract(
         self,
         contract_name: str,
         contract: CompiledContract,
-        constructor_parameters: Tuple[Any, ...] = None,
+        constructor_parameters: Sequence = None,
```

Comment:
Is `Sequence` implicitly `Sequence[Any]`?

Reply:
Yes, the same as with all other types like `Dict` and `Callable`, everything that is not explicitly specified is `Any`.